packages/patternfly-4/react-core/src/components/Dropdown/examples/Dropdown.md

```diff
@@ -780,9 +883,129 @@ class SplitButtonDisabledDropdown extends React.Component {
 }
 ```
 
+```js title=Split-button-action
+import React from 'react';
+import {
+  Dropdown,
+  DropdownToggle,
+  DropdownToggleAction,
+  DropdownItem,
+  DropdownSeparator,
+  DropdownPosition,
+  DropdownDirection,
+  KebabToggle
+} from '@patternfly/react-core';
+import { ThIcon, CogIcon } from '@patternfly/react-icons';
+
+class SplitButtonActionDropdown extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isActionOpen: false,
+      isCogOpen: false
+    };
+    this.onActionToggle = isActionOpen => {
+      this.setState({
+        isActionOpen
+      });
+    };
+    this.onCogToggle = isCogOpen => {
+      this.setState({
+        isCogOpen
+      });
+    };
+    this.onActionClick = event => {
+      console.log('Action clicked!');
+    };
+    this.onCogClick = event => {
+      console.log('Cog clicked!');
+    };
+    this.onActionSelect = event => {
+      this.setState({
+        isActionOpen: !this.state.isActionOpen
+      });
+    };
+    this.onCogSelect = event => {
+      this.setState({
+        isCogOpen: !this.state.isCogOpen
+      });
+    };
+  }
+
+  render() {
+    const { isActionOpen, isCogOpen } = this.state;
+    const dropdownItems = [
+      <DropdownItem key="action" component="button">
+        Action
+      </DropdownItem>,
+      <DropdownItem key="disabled link" component="button" isDisabled>
+        Disabled action
+      </DropdownItem>,
+      <DropdownItem key="other action" component="button">
+        Other action
+      </DropdownItem>
+    ];
+    return (
+      <React.Fragment>
+        <Dropdown
+          onSelect={this.onActionSelect}
+          toggle={
+            <DropdownToggle
+              splitButtonItems={[
+                <DropdownToggleAction
+                  id="example-action-text"
+                  key="action"
+                  aria-label="Action button"
```

Comment:
This button doesn't need an `aria-label` since it has text in it that should describe the action.